lib/worker.js

```diff
@@ -122,6 +127,11 @@ var HTTPListner = function() {
 				site.url = _get['url'];
 				site.file =_get['f'];
 
+				if ( _get['format'] ) {
+					site.format = _get['format'];
+				} else {
+					site.format = DEFAULT_FORMAT;
```

Comment:
Let's just remove the else block and do `site.format = DEFAULT_FORMAT; above the `if`.